addon/components/paper-progress-linear.js

```diff
@@ -60,27 +61,26 @@ export default Ember.Component.extend(ColorMixin, {
   transforms: new Array(101),
 
   setupTransforms() {
-    for (var i = 0; i < 101; i++) {
+    for (let i = 0; i < 101; i++) {
       this.transforms[i] = makeTransform(i);
     }
   },
 
-  bar1Style: Ember.computed('clampedBufferValue', function() {
-    return new Ember.Handlebars.SafeString(this.get('constants.CSS.TRANSFORM') + ': ' + this.transforms[this.get('clampedBufferValue')]);
+  bar1Style: computed('clampedBufferValue', function() {
+    return Ember.String.htmlSafe(this.get('constants.CSS.TRANSFORM') + ': ' + this.transforms[this.get('clampedBufferValue')]);
   }),
 
-  bar2Style: Ember.computed('clampedValue', function() {
+  bar2Style: computed('clampedValue', function() {
 
-    if (this.get('mode') === 'query') {
-      return new Ember.Handlebars.SafeString('');
+    if (this.get('mode') === MODE_QUERY) {
+      return Ember.String.htmlSafe('');
     }
 
-    return new Ember.Handlebars.SafeString(this.get('constants.CSS.TRANSFORM') + ': ' + this.transforms[this.get('clampedValue')]);
+    return Ember.String.htmlSafe(this.get('constants.CSS.TRANSFORM') + ': ' + this.transforms[this.get('clampedValue')]);
   }),
 
-  clampedValue: Ember.computed('value', function() {
-
-    var value = this.get('value');
+  clampedValue: computed('value', function() {
```

Comment:
Here is a much cooler way to clamp values: https://github.com/miguelcobain/ember-paper/blob/wip/v1.0.0/addon%2Fcomponents%2Fpaper-progress-circular.js#L29-L32

I'll make this a computed macro in the future.
Can you change both cps here to clamp using that method?